sphinx/source/docs/examples.rst

```diff
@@ -0,0 +1,20 @@
+:orphan:
+:notoc:
+
+.. _example_index:
+
+Example Index
+=============
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+    :glob:
+
+    examples/plotting
+    examples/models
+
+Blow you can find all standalone examples which are stored in the GitHub repository.
+To see a sublist see :ref:`example_index_models` or :ref:`example_index_plotting`.
+
```

Comment:
For me: typo!

I think here is the place for a explanation why the seperation into `plotting`and `models` exists. What is the idea behind this setup?

Reply:
I'm not sure I understand the question. What does "setup" refer to here, exactly?

Reply:
I am just asking what the difference between "models" and "plotting" is? This is something I don't know and I do not understand why there is this seperation in the folder structure and later in the documentation.